test/parallel/test-tls-securepair-fiftharg.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const tls = require('tls');
+
+const sslcontext = tls.createSecureContext({
+  cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
+  key: fs.readFileSync(common.fixturesDir + '/test_key.pem'),
+});
+
+var catchedServername;
+const pair = tls.createSecurePair(sslcontext, true, false, false, {
+  SNICallback: function(servername, cb) {
```

Comment:
Is it guaranteed that assertion exception will not be hidden? What if `SNIcallback` is called inside `try...catch(e) {}` ?

Anyway, I will fix as you said.